test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -386,35 +385,32 @@ static odp_bool_t approx_eq64(uint64_t val, uint64_t correct)
 static int test_overall_capabilities(void)
 {
 	odp_tm_level_capabilities_t *per_level;
-	odp_tm_capabilities_t        capabilities_array[MAX_CAPABILITIES];
+	odp_tm_capabilities_t        capabilities_array[2];
 	odp_tm_capabilities_t       *cap_ptr;
+	odp_tm_egress_t              egress;
 	uint32_t                     num_records, idx, num_levels, level;
 	int                          rc;
 
-	rc = odp_tm_capabilities(capabilities_array, MAX_CAPABILITIES);
+	odp_tm_egress_init(&egress);
+	egress.egress_kind = ODP_TM_EGRESS_PKT_IO;
+	egress.pktio = xmt_pktio;
+
+	rc = odp_tm_egress_capabilities(&capabilities_array[0], &egress);
```

Comment:
It should be checked that the return value is zero (positive return values are not possible according to the API spec and negatives indicate errors).

Btw, the CU_ASSERT in the existing code below seems to be incorrect (I think the intention is to trigger the assert in the failure case but now it is just dead code).